src/pages/AboutUs/Clients.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { MDBContainer, MDBCol } from 'mdbreact';
+import client1 from './clients/client-1.png';
+import client2 from './clients/client-2.png';
+import client3 from './clients/client-3.png';
+import client4 from './clients/client-4.png';
+import client5 from './clients/client-5.png';
+import client6 from './clients/client-6.png';
+import client7 from './clients/client-7.png';
+import client8 from './clients/client-8.png';
+function Detailsclient() {
+  let clientsLogo = [
+    { client: { logo: client1, name: 'myob company', id: 1 } },
+    { client: { logo: client2, name: 'Belimo company', id: 2 } },
+    { client: { logo: client3, name: 'life groups', id: 3 } },
```

Comment:
I like the fact that you want to show real world logos but since everything is just mock and the logos images in the file structure will just make things messy. I would suggest using mocking images from this server "https://via.placeholder.com/300x150" you can change the width and height by putting any numbers there in place of 300 and the 150 try it